### PR TITLE
Add certificate names for code signing

### PR DESCRIPTION
**Pull request by natemcmaster**
Add configuration for code-signing on binaries and nupkgs.

@jkotalik - I noticed the .nuspec file for Server.IIS isn't actually in use. Okay to remove?

### CODE REVIEW

**Comment by jkotalik** [comment]
Yeah remove the nuspec file. This commit https://github.com/aspnet/IISIntegration/commit/34c797a46b28569415db5f3ab10b2c762489a852 added the csproj to Server.IIS.

File: src/Microsoft.AspNetCore.Server.IIS/Microsoft.AspNetCore.Server.IIS.csproj [hunk 1]
```diff
@@ -23,14 +23,11 @@
...
 
-  <ItemGroup>
-    <SignedPackageFile Include="$(TargetPath)" PackagePath="lib/netstandard2.0/Microsoft.AspNetCore.Server.IIS.dll" Certificate="Microsoft" />
-  </ItemGroup>
-
   <ItemGroup Condition="$(PackNativeAssets) == 'true'">
     <SignedPackageFile Include="aspnetcorev2_inprocess_x86" PackagePath="runtimes/win-x86/nativeassets/$(TargetFramework)/aspnetcorev2_inprocess.dll" Certificate="Microsoft" />
     <SignedPackageFile Include="aspnetcorev2_inprocess_x64" PackagePath="runtimes/win-x64/nativeassets/$(TargetFramework)/aspnetcorev2_inprocess.dll" Certificate="Microsoft" />
 
+    <Content Include="$(PackageId).targets" PackagePath="build/$(TargetFramework)/" />
```
Comment (jkotalik) [inline src/Microsoft.AspNetCore.Server.IIS/Microsoft.AspNetCore.Server.IIS.csproj]: Why was this added?

Comment (natemcmaster) [reply src/Microsoft.AspNetCore.Server.IIS/Microsoft.AspNetCore.Server.IIS.csproj]: The nuspec contained this: https://github.com/aspnet/IISIntegration/blob/182299235438d9272066f0b34fd5f93d24bbe1de/src/Microsoft.AspNetCore.Server.IIS/Microsoft.AspNetCore.Server.IIS.nuspec#L27. Do we still need this targets file? 

https://github.com/aspnet/IISIntegration/blob/182299235438d9272066f0b34fd5f93d24bbe1de/src/Microsoft.AspNetCore.Server.IIS/Microsoft.AspNetCore.Server.IIS.targets#L1-L9

Comment (jkotalik) [reply src/Microsoft.AspNetCore.Server.IIS/Microsoft.AspNetCore.Server.IIS.csproj]: Yeah. Makes sense now 👍
